docs/tutorials/virtuals.md

```diff
@@ -3,13 +3,15 @@
 In Mongoose, a virtual is a property that is **not** stored in MongoDB. 
 Virtuals are typically used for computed properties on documents.
 
-* [Your First Virtual](#your-first-virtual)
-* [Virtual Setters](#virtual-setters)
-* [Virtuals in JSON](#virtuals-in-json)
-* [Virtuals with Lean](#virtuals-with-lean)
-* [Limitations](#limitations)
-* [Populate](#populate)
-* [Further Reading](#further-reading)
+- [Mongoose Virtuals](#mongoose-virtuals)
+  - [Your First Virtual](#your-first-virtual)
+  - [Virtual Setters](#virtual-setters)
+  - [Virtuals in JSON](#virtuals-in-json)
+  - [Virtuals in `console.log()`](#virtuals-in-consolelog)
+  - [Virtuals with Lean](#virtuals-with-lean)
+  - [Limitations](#limitations)
+  - [Populate](#populate)
+  - [Further Reading](#further-reading)
```

Comment:
i dont think this TOC style change is wanted

Reply:
I don't know how this got in there, fixing right now